polar-core/src/bindings.rs

```diff
@@ -425,52 +427,91 @@ impl BindingManager {
     }
 }
 
+fn combine(left: &Symbol, l: Operation, right: &Symbol, r: Operation) -> Operation {
+    let unis = l
+        .constraints()
+        .into_iter()
+        .chain(r.constraints().into_iter())
+        .filter_map(|op| match op {
+            Operation {
+                operator: Operator::Unify,
+                args,
+            } if args.len() == 2
+                && args[0].value().as_symbol().is_ok()
+                && args[1].value().as_symbol().is_ok() =>
+            {
+                Some((
+                    args[0].value().as_symbol().unwrap().clone(),
+                    args[1].value().as_symbol().unwrap().clone(),
+                ))
+            }
+            _ => None,
+        })
+        .chain(std::iter::once((left.clone(), right.clone())));
+    let sub = crate::data_filtering::partition_equivs(unis)
+        .into_iter()
+        .flat_map(|cls| {
```

Comment:
What's `cls` mean? I think `class`? but that doesn't seem right to me

Reply:
I changed the whole way this works, it's a lot simpler now 👍

Reply:
(but it stood for "class" as in "equivalence class")